doc/SecurityBaseline.xml

```diff
@@ -335,12 +349,70 @@
         </thead>
         <tbody valign="top">
           <row>
-            <entry><para>sha256WithRSAEncryption for RSA keys</para></entry>
-            <entry><para>RFC 4055</para></entry>
+            <entry>
+              <para>1.2.840.113549.1.1.11</para>
+            </entry>
```

Comment:
these are signature algorithms can be combined with section 8 signatures.
RSASSA-PSS	1.2.840.113549.1.1.10(OID)

ECDSA 256bit in section can be mapped to ecdsa-with-SHA256 or ecdsa-with-SHA384.